### PR TITLE
make rspec tests pass on windows

### PR DESCRIPTION
**Pull request by zjx20**
better support for windows

### CODE REVIEW

File: lib/fake_dynamo/storage.rb [hunk 1]
```diff
@@ -77,10 +77,11 @@ def load_aof
       log.warn "Loading fake_dynamo data ..."
       loop do
         operation = file.readline.chomp
-        size = Integer(file.readline.chomp)
-        data = file.read(size)
+        size = Integer(file.readline.chomp) - 1
+        data = file.read(size); file.readline 
         db.process(operation, JSON.parse(data))
       end
+      file.close
```
Comment (ananthakumaran) [inline lib/fake_dynamo/storage.rb]: This line will never get executed.

File: lib/fake_dynamo/storage.rb [hunk 2]
```diff
@@ -77,10 +77,11 @@ def load_aof
       log.warn "Loading fake_dynamo data ..."
       loop do
         operation = file.readline.chomp
-        size = Integer(file.readline.chomp)
-        data = file.read(size)
+        size = Integer(file.readline.chomp) - 1
+        data = file.read(size); file.readline
```
Comment (ananthakumaran) [inline lib/fake_dynamo/storage.rb:81]: Is this some window specific quirk. I don't seem to understand why this change is required to make it run in windows.

Comment (zjx20) [inline lib/fake_dynamo/storage.rb:81]: `"\n".bytesize` is returning 1 in windows. However, it will be converted to `"\n\r"` automatically when writing it to file in windows. So we need to read the extra one byte for reading the whole line of data, otherwise it will break up the next operation. But I don't think `data = file.read(size + 1) if windows?` is a good idea. Then I use `readline` to handle it. Let it still have backward compatibility.

File: lib/fake_dynamo/storage.rb (hunk 1 above)
Comment (zjx20) [inline lib/fake_dynamo/storage.rb]: You're right, I will remove this line. Sorry for that.

**Comment by ananthakumaran** [comment]
Thanks
